app.js

```diff
@@ -1,7 +1,18 @@
+const date = new Date();
+document.getElementById('date-time').innerHTML = date;
```

Comment:
- [ ] The current date is displayed below the navigation bar, for this requirement, students should use the  [Luxon](https://moment.github.io/luxon/api-docs/index.html) library. See Get the current date and time
=> This requirement is not fully implemented, the date and time is present but you have to use [Luxon](https://moment.github.io/luxon/api-docs/index.html) as it was required in project requirement, try that out so that it will help you learn more about different libararies